### ISSUE TITLE
iPhone 7 user cannot edit polygons

### ISSUE THREAD

**Issue by wmcclin** [seasketch/next#308]
After submitting a new shape, going back to edit that shape, user clicks on it and it turns yellow. However, she cannot click on and move vertices to change the shape.

**Comment by underbluewaters** [comment]
I don't see any current problem with it on my iPhone. There could be issues with iPhone 7 particularly, but that does seem unlikely. Do we have any confirmation that this was a functional issue with the device and not confusion about the interface?

**Comment by wmcclin** [comment]
No. She watched me go through the process on my iPhone (over Zoom) and repeatedly said, "No, I can't do that. On my phone, the polygon turns yellow but I can't move the vertices."

**Comment by underbluewaters** [comment]
Could be an old version of the operating system. I've installed Sentry.io on the client so we can get error reports and device info. That may help if it happens again.


<img width="1765" alt="Screen Shot 2022-01-25 at 1 23 46 PM" src="https://user-images.githubusercontent.com/511063/151062227-862509be-da67-4585-9414-72220d7bced8.png">
